src/dev.js

```diff
@@ -59,4 +61,10 @@ const matchesNpmWDevScript = function (scriptName, devScriptName) {
 
 const NPM_DEV_SCRIPTS = ['dev', 'serve', 'develop', 'start', 'run', 'build', 'web']
 
+const isExcludedScript = function (scriptValue) {
+  return EXCLUDED_SCRIPTS.some((excluded) => scriptValue.includes(excluded))
```

Comment:
`scriptValue` is validated to be a string in https://github.com/netlify/framework-info/blob/bb9cf27bc7dbe71d06acfb8a110a9dedb23f2881/src/package.js#L38